src/load-components.js

```diff
@@ -1,36 +1,37 @@
 import { maxSatisfying } from 'es-semver';
 
 const CDN_HOST = 'https://components.clever-cloud.com';
+const JS = 'application/javascript';
 const ONE_YEAR = 365 * 24 * 60 * 60;
 
 const MAGIC_MODE_NAME = 'magic-mode';
 const MAGIC_MODE_VALUE = 'dont-use-this-in-prod';
 
-export async function handleLoadRequest ({ getQueryParam, getJson, sendResponse }) {
+export async function handleLoadComponentsRequest (httpApi) {
```

Comment:
Why don't you still use object destructuring?
I am fine with using an `httpApi` variable (I like the naming btw) but maybe you could add some JSDoc

Reply:
I guess it's because I changed the way I call this function, I went from something like this:

```js
handleLoadComponentsRequest({
  getQueryParam (name) { /* ... */ },
  fetch (path, origin, asText = false) { /* ... */ },
  sendResponse (response) { /* ... */ },
});
```

to something like this:

```js
const cloudflareHttpApi = {
  getQueryParam (name) { /* ... */ },
  fetch (path, origin, asText = false) { /* ... */ },
  sendResponse (response) { /* ... */ },
};

handleLoadComponentsRequest(cloudflareHttpApi);
```

It doesn't change anything but I guess I did it because of that.

:+1: I added some JSDoc.

Reply:
I added some JSDoc on the functions.